instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -209,6 +217,10 @@ def __init__(self, *args, **kwargs):
         )
         self._before_request = _before_request
         self.before_request(_before_request)
```

Comment:
Does `_before_request` need the `excluded_urls` passed in as well?

Reply:
Good catch.
I need to extend `test_automatic.py` to cover this case and then I'll push up the change.

Reply:
GitHub didn't update the comment since the change is a few lines up, but this should be taken care of now on line 220 in this commit:

https://github.com/open-telemetry/opentelemetry-python-contrib/pull/604/commits/5996b7e2849b97aa29e4d5d90c95e2b85698feee